src/operators/fsApi/setIdentity.ts

```diff
@@ -0,0 +1,20 @@
+import FSApiOperator from './fsApi';
+
+export default class SetIdentityOperator extends FSApiOperator {
+  // eslint-disable-next-line class-methods-use-this
+  prepareData(inputData:any[]): any[] | null {
+    if (inputData === null || inputData.length < 1) {
+      return null;
+    }
+    const realData:any[] = [
```

Comment:
I think we can consolidate and skip checking the `inputData`. For example, we do `const [uid, properties] = inputData`. And then later use `{uid,properties}` as the second element in `realData`.  This assumes `properties` is allowed to be undefined in the FS API.

Reply:
Or could we do:
```    return [
      'setIdentity',
      { name: inputData[0],
        properties: inputData[1],
       },
    ];
```
As is done elsewhere in the new operators.